Description: intel/xe: skip CLOCK_BOOTTIME when it equals CLOCK_MONOTONIC
 NextBSD, like FreeBSD, defines CLOCK_BOOTTIME as CLOCK_MONOTONIC
 (sys/sys/_clock_id.h), which makes a duplicate case label. Compiled with the
 Intel drivers.
Origin: nextbsd-ports d3479089952d, graphics/mesa-dri/files/patch-src_intel_common_xe_intel__gem.c
 (inherited unchanged from the FreeBSD port graphics/mesa-dri)
Forwarded: no
---
--- a/src/intel/common/xe/intel_gem.c
+++ b/src/intel/common/xe/intel_gem.c
@@ -75,8 +75,10 @@
 #endif
    case CLOCK_REALTIME:
 #ifdef CLOCK_BOOTTIME
+#if !(defined(CLOCK_BOOTTIME) && CLOCK_BOOTTIME == CLOCK_MONOTONIC)
    case CLOCK_BOOTTIME:
 #endif
+#endif
 #ifdef CLOCK_TAI
    case CLOCK_TAI:
 #endif
